compressed_image_message_subscriber_topic: '/camera/front/compressed_image'
image_message_subscriber_topic: '/camera/front/image'

monocular_depth_image_publisher_topic: 'monocular_depth/image'
monocular_metric_depth_image_publisher_topic: 'monocular_metric_depth/image'
monocular_depth_pointcloud_topic: 'monocular_metric_depth/pointcloud'

monocular_depth_pointcloud_frame_id: "map"

depth_service_name: 'depth_anything/depth_service'
metric_depth_service_name: 'depth_anything/metric_depth_service'
metric_pointcloud_service_name: 'depth_anything/metric_pointcloud_service'
metric_depth_with_pointcloud_service_name: 'depth_anything/metric_depth_and_pointcloud_service'

camera_parameters: [234.96454594, 235.1931311, 235.33568193, 205.16684815]  # [Fx, Fy, Cx, Cy]
frame_id: "map"

